torch/csrc/jit/import_legacy.cpp

```diff
@@ -11,7 +11,7 @@
 
 #include "caffe2/core/common.h"
 #include "caffe2/core/types.h"
-#include "caffe2/proto/caffe2_pb.h"
+#include <caffe2/proto/caffe2_pb.h>
```

Comment:
Why? This shouldn't be needed -- it's not a system include.

Reply:
I was trying to remove this include to resolve a tidy issue and added it back because some caffe2 test failure. I'll change the style back. Thanks!